docs-source/mbaas/content/setup/_index.md

```diff
@@ -64,25 +64,47 @@ To start the installation, take the following steps:
 11. In the **Database Options** section, you can customize the database shape and the CIDR for client access. Note that you cannot access
     **Database Actions** if you change the network access to `PRIVATE_ENDPOINT_ACCESS`.
 
-12. Once you have completed customization, click **Next**. 
+12. Once you have completed customization, click **Next**.
 
 13. The **Review** page is displayed. Check your settings and then click **Create** to create the stack and run the Terraform `apply`
-    command to create all of the associated resources.  
+    command to create all of the associated resources.
 
 You can monitor the installation in the log. Installation takes approximately 20 minutes to complete.  Most of this time is spent provisioning
 the Kubernetes cluster, its nodes, and the database.
 
 When the installation is finished, some important information is included at the end of the log.  You need this information to access
 the newly created environment. For example:
 
+```text
+kubeconfig_cmd = "oci ce cluster create-kubeconfig --cluster-id ocid1.cluster.oc1.iad.aaaaaaaatc --region us --token-version 2.0.0 --kube-endpoint PUBLIC_ENDPOINT --file $HOME/.kube/config"
+parse_application_id = "MYCOOLAPP001"
+parse_dashboard_password = <sensitive>
+parse_dashboard_uri = "https://1.2.3.4/parse-dashboard"
+parse_dashboard_user = "ADMIN"
+parse_endpoint = "https://1.2.3.4/parse"
+parse_master_key = <sensitive>
 ```
-application_id = "COOLAPPV100"
-dashboard_password = <sensitive>
-dashboard_uri = "http://1.2.3.4"
-dashboard_user = "ADMIN"
-kubeconfig_cmd = "oci ce cluster create-kubeconfig --cluster-id ocid1.cluster.oc1.iad.xxx
- --file $HOME/.kube/config --region us-ashburn-1 --token-version 2.0.0 --kube-endpoint PUBLIC_ENDPOINT"
-parse_endpoint = "1.2.3.4/parse"
-```
+
+## TLS
+
+The Oracle Backend for Parse Platform is deployed with a sample self-signed certificate for TLS.  This results in an "Accept Risk" message when accessing the Parse Dashboard and the sample TLS certificate should not be used for production deployments.
```

Comment:
In the first sentence, change "TLS" to "Transport Layer Security (TLS)".

It looks like there may be an extra space between sentences. There should only be one space between sentences.